LD_Python_Examples.ipynb

```diff
@@ -1972,6 +1972,13 @@
     "draw_all_lds(LD_forward, LD_backward, grid_parameters, tau, p_value)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "draw_all_lds returns the handles of the figures and corresponding axes that can be further customised as any other matplotlib plot"
```

Comment:
Not clear how to customise a figure using the plot handle.
Add a concrete example or at least a link to a worked example shown somewhere else in "HOW-TOs" section of Notebook.